tests/pipelines/test_pipelines_automatic_speech_recognition.py

```diff
@@ -53,7 +53,7 @@
 # from .test_pipelines_common import CustomInputPipelineCommonMixin
 
 
-class AutomaticSpeechRecognitionPipelineTests(unittest.TestCase, metaclass=PipelineTestCaseMeta):
+class AutomaticSpeechRecognitionPipelineTests(unittest.TestCase):
```

Comment:
all changes in `tests/pipelines/test_pipelines_xxx.py` are just removing the usage of `PipelineTestCaseMeta`, no need to go through all of them